src/main/java/Duke.java

```diff
@@ -1,10 +1,268 @@
+                }
+            }
+            else if(deadline){
+                if(order.length() > 8){
+                    System.out.println("Ka To:");
+                    System.out.println();
+                    int number = firstSlashTracker(order);
+                    System.out.println("Yup, this is now added! " + " [D][x] " +  order.substring(8, number) + "(by: " +
+                            order.substring(number + 3) + ")");
+                    int size = orders.size() + 1;
+                    System.out.println(" You have " + size + " task added now! ");
+                }
+            }
+            else if(event){
+                if(order.length() > 5){
+                    System.out.println("Ka To:");
+                    System.out.println();
+                    int number = firstSlashTracker(order);
+                    System.out.println(" Yup, this is now added! " + " [E][x] " +  order.substring(5, number) + "(at: " +
+                            order.substring(number + 3) + ")");
+                    int size = orders.size() + 1;
+                    System.out.println(" You have " + size + " task added now! ");
+                }
+            }
+            else {
+                correct = false;
+                System.out.println("Ka To:");
+                System.out.println(" ");
+                System.out.println("Sorry, I could not answer that");
+            }
+            if(correct && !done && !deleted && !order.equals("task list")) {
+                if(todo){
+                    if(order.length() <= 4){
+                        System.out.println("oops, the todo description cannot be empty");
+                    }else {
+                        Task item = new Task(0, 0, order.substring(4) );
+                        orders.add(item);
+                    }
+                }else if(deadline){
+                    if(order.length() <= 8){
+                        System.out.println("oops, the deadline description cannot be empty");
+                    }else {
+                        int number = firstSlashTracker(order);
+                        Task item = new Task(1, 0,order.substring(8, number) + "(by: " +
+                                order.substring(number + 3) + ")");
+                        orders.add(item);
+                    }
+                }else if(event){
+                    if(order.length() <= 5){
+                        System.out.println("oops, the event description cannot be empty");
+                    }else {
+                        int number = firstSlashTracker(order);
+                        Task item = new Task(2, 0, order.substring(5, number) + "(at: " +
+                                order.substring(number + 3) + ")");
+                        orders.add(item);
+                    }
+
+                }else {
+                    Task item = new Task(4, 0, order);
+                    orders.add(item);
+                }
+
+            }
```

Comment:
I'm not sure why you decided to make another if block to check for todo, event, and deadline since you already checked for the booleans at lines 117, 126 and 137. Would combining the bodies of the 2 if blocks of todo change the output in any way?